hypha/settings/base.py

```diff
@@ -562,6 +562,12 @@
 # Finance extension to finance2 for Project Invoicing
 INVOICE_EXTENDED_WORKFLOW = env.bool("INVOICE_EXTENDED_WORKFLOW", True)
 
+# Application Preview Settings
+
+# Require an applicant to view their rendered application before submitting
+REQUIRE_PREVIEW = env.bool("REQUIRE_PREVIEW", True)
```

Comment:
1. Can we move this up to the "# Hypha custom settings" section. Good to have all the custom stuff in one place I think.
2. The setting name is a bit terse, could we do something like `SUBMISSION_PREVIEW_REQUIRED`? Something that tell what part of Hypha the setting refers to.